lblod/frontend-lokaal-mandatenbeheer
# Mayor selector

## Description

We want to be able to select a mayor in the "Voorbereiding legislatuur"

## How to test

- If there is no mayor selected yet, try selecting a person that should be mayor
- If there already is a mayor selected, try changing the mayor with the person selector
- If there are multiple mayors for this "bestuursperiode", then an error message should be shown (more mayors can be created through the bestuursorgaan route)

![image](https://github.com/lblod/frontend-lokaal-mandatenbeheer/assets/55446974/908c6629-dc1e-4f6a-90e3-f8e8dbbc125b)
